Majasdarbs/03-majasdarbs/Gvido-CV-CSS/styles.css

```diff
@@ -0,0 +1,149 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import "responsive.css";
+
+* {
+    margin: 0;
+}
+
+body {
+    background-color: rgb(0, 0, 0);
+    color: white;
+    font-family: 'Roboto', 'Arial', 'sans-serif'; 
+}
+
+/* Navigation bar >>>>>>>>>>>>>>>>>>>>>*/
+.nav-bar {
+    display: flex;
+    justify-content: center;
+    background-color: white;
+    font-size: 20px;
+    position: fixed;
+    width: 100vw;
+    top: 0;
+}
+
+.nav-bar a{
+    text-decoration: none;
+    color: black;
+    padding: 0.3rem;
+    margin-left: 10px;
+}
+
+.nav-bar a:hover {
+    color: rgb(255, 255, 255);
+    padding: 1rem;
+    background-color: rgb(0, 0, 0);
+    transition: 0.7s;
+}
+
+/* Header >>>>>>>>>>>>>>>>>*/
+
+/* ABOUT ME */
+#about-me {
+    margin-top: 2rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
```

Comment:
if you are using rem, add font-size to the same place you declare font-family.
It will work like this:
html {
font-size: 10px;
font-family: welofn23ioh2i3
}
1rem = 10px